substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfDebugInfo.java

```diff
@@ -51,9 +51,9 @@
     public static final String DW_ARANGES_SECTION_NAME = ".debug_aranges";
 
     /**
-     * Currently generated debug info relies on DWARF spec vesion 2.
+     * Currently generated debug info relies on DWARF spec version 2.
      */
-    public static final short DW_VERSION_2 = 2;
+    public static final short DW_VERSION = 2;
```

Comment:
We might need to consider bumping this to 3 now that we are using `DW_AT_call_line` and `DW_AT_call_file` that are not defined in version 2 of DWARF.

Note: gdb doesn't really mind and happily consumes them even with dwarf-v2